knex/seeds/05_annotations.js

```diff
@@ -0,0 +1,42 @@
+exports.seed = (knex) => {
+  const dropFk = 'ALTER TABLE "Annotations" DROP CONSTRAINT IF EXISTS "Annotations_fk0"; ALTER TABLE "Annotations" DROP CONSTRAINT IF EXISTS "Annotations_fk1"; ALTER TABLE "Annotations" DROP CONSTRAINT IF EXISTS "Annotations_fk2";';
```

Comment:
if you put backticks you could spread this over a few lines and make it more readable 👍